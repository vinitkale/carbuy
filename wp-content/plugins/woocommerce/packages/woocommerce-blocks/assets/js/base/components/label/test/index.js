version https://git-lfs.github.com/spec/v1
oid sha256:147b867d77f13ac0872d740e48a0bf5c973001fa7a4fbbd701032e07c60a0aa5
size 1880
