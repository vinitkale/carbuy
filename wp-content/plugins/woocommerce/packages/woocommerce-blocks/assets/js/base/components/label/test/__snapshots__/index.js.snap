version https://git-lfs.github.com/spec/v1
oid sha256:a6230deecbba1409725d4a38e44ce0e1b1da9380098bf2748777918a8af3464d
size 1101
